packages/editor-ui/src/components/Node.vue

```diff
@@ -778,6 +826,45 @@ export default defineComponent({
 		}
 	}
 
+	&.touch-active,
+	&:hover,
+	&.menu-open {
+		.node-options {
+			opacity: 1;
+		}
+	}
+
+	.node-options {
+		:deep(.button) {
+			--button-font-color: var(--color-text-light);
+			--button-border-radius: 0;
+		}
+		cursor: default;
+		position: absolute;
+		top: -34px;
+		z-index: 11;
+		min-width: 100%;
+		display: flex;
+		left: calc(-1 * var(--spacing-4xs));
+		right: calc(-1 * var(--spacing-4xs));
+		justify-content: center;
+		align-items: center;
+		padding-bottom: var(--spacing-2xs);
+		font-size: var(--font-size-s);
+		opacity: 0;
+		transition: opacity 100ms ease-in;
+
+		button {
+			background-color: var(--color-canvas-background);
```

Comment:
Nitpicking :P but couldn't we apply this only to the `&-inner` container? This way we wouldn't have to style inner elements of a component and hence it would be less fragile in-case layout of the component changes in the future